src/Azure.Functions.Cli/Common/Constants.cs

```diff
@@ -78,6 +79,14 @@ public static class KuduLiteDeploymentConstants
         {
             public const int ArmTokenExpiryMinutes = 4;
             public const int StatusRefreshSeconds = 3;
+
+            public static readonly IDictionary<string, string> LinuxDedicatedBuildSettings = new Dictionary<string, string>
+            {
+                { "ENABLE_ORYX_BUILD", "true" },
+                { "SCM_DO_BUILD_DURING_DEPLOYMENT", "1" },
+                { "BUILD_FLAGS", "UseExpressBuild" },
+                { "XDG_CACHE_HOME", "/tmp/.cache" }
```

Comment:
Had to make this point to a local dir to optimize build times on Elastic Premium plans. I will try and change it to `/var` instead after testing it a bit to make sure temp cleanups don't affect the builds.